src/quadrature/Galerkin_Ordinate_Space.cc

```diff
@@ -285,7 +280,8 @@ void Galerkin_Ordinate_Space::compute_operators() {
   // fill cartesian_ordinates
   unsigned count(0);
   for (unsigned i = 0; i < numOrdinates; ++i) {
-    if (ordinates[i].wt() != 0) {
+    if (std::abs(ordinates[i].wt()) >
+        std::numeric_limits<decltype(ordinates[i].wt())>::min()) {
```

Comment:
Do we have plans to wrap these comparisons in a draco class at some point? I really like the use of the type-specific min(), for instance, but the full syntax is pretty intense!